lib/websocket.js

```diff
@@ -144,7 +144,7 @@ function init (env, ctx, server) {
           socket.emit('dataUpdate',lastData);
         }
         console.log('Authetication request from client ID: ',socket.client.id);
-        callback(socketAuthorization);
+        if (callback) callback(socketAuthorization);
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '{' and instead saw 'callback'.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1655370455/issues/source?bid=2452379&fileBranchId=3037395#l147)